### PR TITLE
Gatsby Gazette October

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
Ghost completing this for @m-allanson who wrote all of this. Come for the jokes, stay for the great updates.

### CODE REVIEW

File: docs/blog/2019-11-11-gazette-october/index.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Have you ever wondered what Gatsby _does_ when you run a build? Now there's a new docs page that runs through the details. [Overview of the Gatsby Build Process](https://www.gatsbyjs.org/docs/overview-of-the-gatsby-build-process/) is the place to go. I particularly like [the diagram that explains the different layers of Gatsby](https://www.gatsbyjs.org/docs/overview-of-the-gatsby-build-process/#what-happens-when-you-run-gatsby-build).
+
+### Schema Customization Examples
+
+We're gradually moving official plugins over to use the schema customisation APIs added earlier this year. See [the changes to `gatsby-transformer-sharp` for an example](https://github.com/gatsbyjs/gatsby/pull/18871). If you're interested in getting started with this, there's a little-known plugin that will write out the schema for your inferred data. See https://www.gatsbyjs.org/packages/gatsby-plugin-schema-snapshot/
+
+### Offline plugin
+
+The [offline plugin has been updated to handle more edge cases](https://github.com/gatsbyjs/gatsby/pull/17590). Now the plugin will revert back to the network if a visitor disables JavaScript after the site has already been cached. Thanks to @davidbailey00.
```
Comment (fk) [inline docs/blog/2019-11-11-gazette-october/index.mdx]: Not sure if we usually do this or not, but consider linking @davidbailey00 to the GitHub profile?

Comment (sidharthachatterjee) [reply docs/blog/2019-11-11-gazette-october/index.mdx]: Yup, we usually _do_ that. This was oversight! Thanks, Flo ❤️

File: docs/blog/2019-11-11-gazette-october/index.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## Theme of the Month
+
+![screenshot of gatsby-theme-firebase's readme](./gatsby-theme-firebase.png)
+
+[`gatsby-theme-firebase`](https://github.com/epilande/gatsby-theme-firebase) might be the quickest way to add full-featured auth to your site, and it's our theme of the month for this month!
+
+Add the theme, your Firebase credentials and off you go. Want to change anything? Use shadowing to override whichever components you want.
+
+## How to Equip your Marketing Team for Success
```
Comment (wardpeet) [inline docs/blog/2019-11-11-gazette-october/index.mdx]: We probably want to update this 😄 as it's outdated

Comment (sidharthachatterjee) [reply docs/blog/2019-11-11-gazette-october/index.mdx]: Yeah, I’m getting this in today. Will update this with a section about Gatsby Cloud

**Comment by muescha** [comment]
no mention of Gatsby Cloud?

File: docs/blog/2019-11-11-gazette-october/index.mdx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+date: 2019-11-11
+author: Mike Allanson
+excerpt: "It's November already. But what happened to October? Read more to find out."
+tags:
+  - gazette
+---
+
+October means halloween! For your spooky reading pleasure the blog now has a dark mode. Dim the lights, pull up a ocmfy chair and we can begin.
+
+In September we posted our [Series A funding announcement](https://www.gatsbyjs.org/blog/2019-09-26-announcing-gatsby-15m-series-a-funding-round/), then headed to the UK for Gatsby Days London. Fear not, if you missed all the action you can [catch up by watching the videos](https://www.youtube.com/watch?v=UvO7h_KwgpE&list=PLCU2qJekvcN2f9rgS50xy7-4QBpgwid4G).
```
Comment (muescha) [inline docs/blog/2019-11-11-gazette-october/index.mdx]: removing domain only in docs or also in blog posts?

```suggestion
In September we posted our [Series A funding announcement](/blog/2019-09-26-announcing-gatsby-15m-series-a-funding-round/), then headed to the UK for Gatsby Days London. Fear not, if you missed all the action you can [catch up by watching the videos](https://www.youtube.com/watch?v=UvO7h_KwgpE&list=PLCU2qJekvcN2f9rgS50xy7-4QBpgwid4G).
```

Comment (sidharthachatterjee) [reply docs/blog/2019-11-11-gazette-october/index.mdx]: More relevant for docs, this is fine, I think. Good catch though!

**Pull request merged**